.github/workflows/commit-check.yaml

```
@@ -49,9 +57,30 @@ jobs:
             MESSAGE=$(git log --format="%s%n%n%b" -n 1 $commit)
             
             # Check commit message format
-            if [[ ! $(echo "$MESSAGE" | head -n 1) =~ $ALLOWED_KEYWORDS ]]; then
+            if [[ ! "$MESSAGE" =~ $COMMIT_PATTERN ]]; then
               COMMENT_BODY+=$(printf -- "- **Invalid commit message format (Commit ID: %s):**\n\`%s\`\n\n" "$commit" "$MESSAGE")
               INVALID_COMMITS=$((INVALID_COMMITS+1))
+            else
+              COMMIT_KEYWORD="${BASH_REMATCH[1]}"
+              COMMIT_SCOPE="${BASH_REMATCH[2]}"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)